content/accessibility/general-overview.mdx

```diff
@@ -0,0 +1,116 @@
+# General guidelines
```

Comment:
I'm unsure about this page, I think if it's called _general_ guidelines maybe it would be easier to link to an existing checklist, does MDN have a good list? Or let's make it more specific. Right now the page isn't easy to read because it jumps off to other pages many times. 
I'm wondering if we should make the content here more concise with: color, typography, alt text (and generally using visuals), keyboard navigation, and content, as the key areas a designer needs to think about when designing accessibly, with a few bullet points for each. The W3C quick reference can be at the end of the list, and maybe it should all go in the designer tooklit page? Happy to discuss further.